src/main/java/org/folio/services/invoice/BaseInvoiceService.java

```diff
@@ -194,4 +195,12 @@ private boolean recalculateInvoiceTotals(Invoice invoice, List<InvoiceLine> line
     calculateTotals(invoice, lines);
     return !Objects.equals(adjustmentsTotal, invoice.getAdjustmentsTotal());
   }
+
+  public Future<Void> updateVoucherNumberInInvoice(Voucher voucher, RequestContext requestContext) {
+    return getInvoiceById(voucher.getInvoiceId(), requestContext)
+      .compose(invoice -> {
+        invoice.setVoucherNumber(voucher.getVoucherNumber());
```

Comment:
lets add log 
`log.info("updateVoucherNumberInInvoice:: VoucherNumber '{}' was set to invoice '{}', voucher.getVoucherNumber, invoice.getId)

Reply:
sure, Will do. sorry I missed this message.